core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -925,8 +933,10 @@ else
 
   if [ -f $MY_LOCAL_SW_DIST/${HADOOP_TAR} ]; then
     cp $MY_LOCAL_SW_DIST/${HADOOP_TAR} .
+    echo "Downloaded Hadoop tar file: $MY_LOCAL_SW_DIST/${HADOOP_TAR}" 
```

Comment:
Given that we didn't download anything, should this message say "copied" instead of "downloaded"? Same for the following ones.